Libraries/Core/NativeExceptionsManager.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+export type StackFrame = {|
+  column: ?number,
+  file: string,
+  lineNumber: number,
+  methodName: string,
+|};
+
+export interface Spec extends TurboModule {
+  +reportFatalException: (
+    message: string,
+    stack: Array<StackFrame>,
+    exceptionId: number,
+  ) => void;
+  +reportSoftException: (
+    message: string,
+    stack: Array<StackFrame>,
+    exceptionId: number,
+  ) => void;
+  +updateExceptionMessage: (
+    message: string,
+    stack: Array<StackFrame>,
+    exceptionId: number,
+  ) => void;
+  // Android only
+  +dismissRedbox?: () => void;
```

Comment:
@RSNara  🤔 Seems `sentIntent` of `IntentAndroid` not marks optional
https://github.com/facebook/react-native/pull/24877/files#diff-7033e846872152e52446abb20da3efb8R25, should we also changed that?

Reply:
Yeah, nice catch. We should change that.

Reply:
Actually, no. I was wrong earlier about our codegen output. Turns out, regardless of if a method is optional or not, it'll generate a required method in native. So, making this optional doesn't do anything.

Reply:
👌 Get it.